kairon/shared/chat/processor.py

```
@@ -353,6 +355,16 @@ def validate_media_file_type(bot: str, file_content: File):
-            raise AppException(f"File '{file_content.filename}' already exists. Please upload a different file.")
\ No newline at end of file
+        user_media_data_obj = UserMediaData.objects(
+            bot=bot,
+            filename=file_content.filename
+        ).order_by('-timestamp').first()
+
+        if user_media_data_obj:
+            if user_media_data_obj.timestamp >= thirty_days_ago:
+                raise AppException(
+                    f"File '{file_content.filename}' already exists. Please upload a different file."
+                )
```

Ruff (0.15.4)

[warning] Avoid specifying long messages outside the exception class

(TRY003)